db/schema.rb

```diff
@@ -98,15 +91,6 @@
     t.index ["vendor_id"], name: "index_vendor_ownerships_on_vendor_id"
   end
 
-  create_table "vendor_tags", force: :cascade do |t|
```

Comment:
Same with this table, why is this being deleted?

Reply:
It seems these were removed in the new migration. David says these tables should have been removed a while ago, as they are not in use anymore. So this should be fine.